lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -482,6 +476,96 @@ def operations
       end
     end # End of POLICY_ELEMENT_TYPES iteration
 
+    # Given a policy element class and a set of conditions, returns an
+    # ActiveRecord_Relation with those conditions applied
```

Comment:
* `ActiveRecord::Relation`

Reply:
Weirdly enough, the class that is returned is `ActiveRecord_Relation`

Reply:
Oh lol okay.